src/cpp/session/modules/SessionPackages.R

```diff
@@ -1086,10 +1123,11 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
    if (grepl("MIT\\s+\\+\\s+file\\s+LICEN[SC]E", DESCRIPTION$License, perl = TRUE))
    {
       # Guess the copyright holder
-      holder <- if (!is.null(getOption("devtools.name")))
-         Author
-      else
+      holder <- .rs.nullCoalesce(
+         getOption("usethis.full_name"),
```

Comment:
FWIW usethis doesn't explicitly support anything around this and, instead, would expect/hope a user to place the license with `use_mit_license()`.

Reply:
Makes sense -- I'll keep this in any case since RStudio is already trying to generate the LICENSE file in this scenario.